### ISSUE TITLE
Behavior Bug: Instagram behaviour only opens the first post of the row

### ISSUE THREAD

**Issue by nvanderperren** [webrecorder/browsertrix-behaviors#58]
**URL**
[https://www.instagram.com/elsdietvorst18](https://www.instagram.com/elsdietvorst18)

**Describe the bug**
Instagram behaviour only opens the first post of the row and ignores the two other posts. When using archiveweb with autopilot, only the first post of the row is archived. You can also see it in the stats. The number of captured posts is the same as the numer of captured rows. So you miss 2/3 of all posts when using Autopilot

**What tool do you see this issue in?**
- [x] ArchiveWeb.page Extension
- [ ] ArchiveWeb.page App
- [ ] Browsertrix Crawler
- [ ] Other/custom

**Screenshots**
<img width="401" alt="Screenshot 2023-08-31 at 14 18 13" src="https://github.com/webrecorder/browsertrix-behaviors/assets/17183054/0e69b91d-7d32-4d96-bb90-562b0337869e">

**Additional context**
n/a

**Comment by edsu** [comment]
I just ran it (chrome extension v0.11.2) and it seemed to complete the rows fine. Note I did wait for the initial page to load fully (all green) before starting autopilot, instead of starting with autopilot (see https://github.com/webrecorder/archiveweb.page/issues/185)

<img width="1484" alt="Screenshot 2023-09-04 at 5 14 33 PM" src="https://github.com/webrecorder/browsertrix-behaviors/assets/33829/ab9d3feb-90a4-4bdd-8618-6bdbd9e0e98d">

You can browse the complete archive here (temporarily) to see if it looks complete?

https://replayweb.page/?source=https%3A%2F%2Fedsu-webarchives.s3.amazonaws.com%2Ftmp%2Felsdievorst18-instagram.wacz#view=pages

**Comment by nvanderperren** [comment]
it seems that your's is complete, but I keep having this issue. I do everything the same as you do.

**Comment by nvanderperren** [comment]
sorry for the late answer. you can find the video here. As you can see: 3 posts and 3 rows, so one post per row. 

https://github.com/webrecorder/browsertrix-behaviors/assets/17183054/e5755938-f157-4bc0-b201-c992a5b15a8a

**Comment by edsu** [comment]
This is a long shot, but does it help if you disable your other browser plugins?

**Comment by ikreymer** [comment]
May be fixed with latest instagram behavior, needs testing again.

**Comment by Shrinks99** [comment]
@Shrinks99 Test this

**Comment by Shrinks99** [comment]
Tested this today, was unable to reproduce the bug, autopilot correctly captured the posts in the correct order.

**Comment by ikreymer** [comment]
This appears to be fixed, please open a new issue if Instagram behavior breaks again.
